drivers/i3c/master/mipi-i3c-hci/core.c

```diff
@@ -196,6 +201,11 @@ static void aspeed_i3c_of_populate_bus_timing(struct i3c_hci *hci, struct device
 	/* core_period is in nanosecond */
 	core_period = DIV_ROUND_UP(1000000000, core_rate);
 
+	if (!i3c_device_power) {
+		hci->master.bus.scl_rate.i3c = 1000000;
```

Comment:
Let's make sure this is temporary. It should not be carried over, as it defeats I3c mode switch from open drain (1 MHZ) to push pull mode (>1 MHZ to 12.5 MHz)

Reply:
> Let's make sure this is temporary. It should not be carried over, as it defeats I3c mode switch from open drain (1 MHZ) to push pull mode (>1 MHZ to 12.5 MHz)

Added comments in the code.